third_party/proton/test/test_profile.py

```diff
@@ -197,7 +197,33 @@ def foo(x, size: tl.constexpr, y):
         assert data[0]["children"][0]["frame"]["name"] == "test0"
         assert data[0]["children"][0]["children"][0]["frame"]["name"] == "foo_test_1ctas_1elems"
         assert data[0]["children"][0]["children"][0]["metrics"]["flops32"] == 1.0
-        assert data[0]["children"][0]["children"][0]["metrics"]["Time (ns)"] > 0
+        assert data[0]["children"][0]["children"][0]["metrics"]["time (ns)"] > 0
+
+
+def test_pcsampling():
```

Comment:
Maybe explicitly test kernels with and without line info?

Reply:
Updated